core/src/main/resources/jenkins/model/Jenkins/manage.jelly

```diff
@@ -68,7 +69,7 @@ THE SOFTWARE.
             </j:when>
             <j:when test="${m.requiresPOST}">
               <f:link href="${m.urlName}" post="${m.requiresPOST}">
-                <img class="icon" src="${iconUrl}" alt="manage-option"/>
+                <img class="icon" src="${iconUrl}" alt="[${%alt}]"/>
```

Comment:
Given to the size of the images is known, I would add the attributes `width` and `height`.  [Ref](https://developer.mozilla.org/en-US/docs/Learn/HTML/Multimedia_and_embedding/Images_in_HTML)